ac_runtime-api/modules/nominationpools.adoc

```diff
@@ -0,0 +1,16 @@
+[#sect-runtime-nomination-pools-module]
+=== Module Nomination Pools
+
+IMPORTANT: This section describes *Version 1* of this API. Please check `Core_version` (<<defn-rt-core-version>>) to ensure compatibility. Currently supports only one RPC endpoint. 
+
+
+[#sect-rte-pending-rewards]
+==== `NominationPools_pending_rewards`
```

Comment:
* The `NominationPools` prefix should be `NominationPoolsApi`.
* It looks like the `points_to_balance` and `balance_to_points` APIs are missing?

Reply:
Fixed the first type. Regarding second, `points_to_balance` and `balance_to_points` are not declared as Runtime APIs.